index.html

```diff
@@ -354,7 +348,7 @@ <h3 id="string_specific_language">String-specific language information</h3>
 
 <p class="advisement" id="bp-lang_field_based_metadata">Use field-based metadata or string datatypes to indicate the language and the base direction for individual natural language strings.</p>
 
-<p>There is widespread low-level support for natural language string metadata because the use of metadata for storage and interchange of the language of data values is long-established and widely supported in the basic infrastructure of the Web. This includes language attributes in [[XML]] and [[HTML]]; string types in schema languages (e.g. [[xmlschema11-2]]) or the various RDF specifications including [[JSON-LD]]; or protocol- or document format-specific provisions for language.</p>
+<p> Low-level support for natural language string metadata is widespread because the use of metadata for storage and interchange of the language of data values is long-established and widely supported in the basic infrastructure of the Web. This includes language attributes in [[XML]] and [[HTML]]; string types in schema languages (e.g. [[xmlschema11-2]]) or the various RDF specifications including [[JSON-LD]]; or protocol- or document format-specific provisions for language.</p>
```

Comment:
Sorry to be a bit picky... 

> various RDF specifications including [[JSON-LD]]

is not really precise. As far as the core RDF is concerned there is only one (family) of spec; in this case the reference should probably be `https://www.w3.org/TR/rdf11-concepts/`. Then there are several specification for the _serialization_ of the general RDF concepts, of which JSON-LD is one. The example that you give below is in JSON-LD; it may be worth to note, then, that in this document the JSON-LD serialization is used for the examples, but it also applies to, e.g., Turtle.

Reply:
Oh yes, there are many edits to be made relating to the current state, mostly in the section about best practices but also elsewhere, which need to be made.  Addison has an action to work on that, and will hopefully take your comment into account.  I'm not really intending to touch that stuff - i was just improving the english.